build-support/bin/_release_helper.py

```diff
@@ -903,49 +903,6 @@ def publish() -> None:
     prompt_to_generate_docs()
 
 
-def publish_apple_silicon() -> None:
```

Comment:
Seems unrelated?

Reply:
Yeah:

> Also remove Apple Silicon code from our release script

 didn't seem like it needed a dedicated PR. I can split if you prefer